project/src/main/java/com/google/sps/Models/Database.java

```diff
@@ -419,4 +419,11 @@ private static ArrayList getListProperty(Entity entity, String prop) {
     }
     return commentObjects;
   }
+
+  // Deletes a document
+  public static void deleteDocument(String hash) {
```

Comment:
I feel like you should also take care of deletions in the user's documents list and the folder documents list

Reply:
yea the feature isnt done yet, thats why I didn't request any reviewers yet lol